cli/main.go

```diff
@@ -36,22 +36,29 @@ func usage() {
 	fmt.Fprintf(os.Stderr, "    %s [options] MOUNTPOINT PROJECT1 PROJECT2 ...\n", progName)
 	fmt.Fprintf(os.Stderr, "    %s [options] MOUNTPOINT manifest.json\n", progName)
 	fmt.Fprintf(os.Stderr, "options:\n")
-	flag.PrintDefaults()
+	// Hide experimental options
+	flag.VisitAll(func(f *flag.Flag) {
+		if f.Name == "readWrite" || f.Name == "daemon" {
+			return
+		}
+		name, usage := flag.UnquoteUsage(f)
+		fmt.Fprintf(os.Stderr, "  -%s %s \n\t%s\n", f.Name, name, usage)
+	})
 	fmt.Fprintf(os.Stderr, "\n")
 	fmt.Fprintf(os.Stderr, "A project can be specified by its ID or name. The manifest is a JSON\n")
 	fmt.Fprintf(os.Stderr, "file describing the initial filesystem structure.\n")
 }
 
 var (
 	debugFuseFlag = flag.Bool("debugFuse", false, "Tap into FUSE debugging information")
-	daemon = flag.Bool("daemon", false, "An internal flag, do not use it")
-	fsSync = flag.Bool("sync", false, "Sychronize the filesystem and exit")
-	gid = flag.Int("gid", -1, "User group id (gid)")
-	help = flag.Bool("help", false, "display program options")
-	readOnly = flag.Bool("readOnly", false, "mount the filesystem in read-only mode")
```

Comment:
Can we keep -readOnly for backwards compatibility (it will do nothing as this is the default, but can not be specified together with -readWrite)?

Reply:
Updated with `-readOnly` still accepted, but does not change behavior as that's now the default.